stories/dooboo-ui/NetworkImageStories/index.tsx

```diff
@@ -20,38 +21,34 @@ function NetworkImageStory(): React.ReactElement {
       }}
     >
       <NetworkImage
-        style={{
-          margin: 20,
-          width: 400,
-          height: 300,
-        }}
-        styles={{image: {width: '50%'}}}
-        url="https://reactnative.dev/img/tiny_logo.png"
-      />
-      <NetworkImage
-        style={{
-          margin: 20,
-          width: 180,
-          height: 180,
-        }}
-        url="https://wronglink.co"
-      />
-      <NetworkImage
-        style={{
-          margin: 20,
-          width: 180,
-          height: 180,
-        }}
-        url="https://media.vlpt.us/images/luck2901/post/5745952f-eb96-4784-b01c-2eb90158ace7/React_Native_Tutorial.jpg"
-      />
-      <NetworkImage
-        style={{
-          margin: 20,
-          width: 180,
-          height: 180,
-        }}
-        url="https://upload.wikimedia.org/wikipedia/commons/6/69/Very_Large_Telescope_Ready_for_Action_%28ESO%29.jpg"
+        style={{margin: 40}}
+        styles={{image: {width: 300, height: 300}}}
+        url="wrong-link"
       />
+
+      <View style={{width: 400, height: 300, margin: 20}}>
+        <NetworkImage
+          styles={{image: {flex: 1, alignSelf: 'stretch'}}}
+          url="https://reactnative.dev/img/tiny_logo.png"
+        />
+      </View>
```

Comment:
I understand your point. 

I think `Image` and `loadingSource` should be wrapped with same `View`, because they share same center point. Otherwise It is hard to be done.

Plus, when both `Image` and `loadingSource` is not wrapped with `View`, `loadingSource` resizes to `Image` just before loading finishes.

Uploading 화면 기록 2021-09-22 오전 11.53.39.mov…
(When I wrap `Image` or `loadingSource` with empty `View`, it is fine, but it is unnatural.)


So it is hard to remove parent `View`. I think with `style` and `styles.image`.

Do you have any solutions?

Reply:
Then how about adding another container `View`?

```
<View style={{width: 400, height: 300, margin: 20}}>
  <NetworkImage
    styles={{image: {flex: 1, alignSelf: 'stretch'}}}
    url="https://reactnative.dev/img/tiny_logo.png"
  />
</View>
```
Bringing everything into `NetworkImage`. Do you thing that would work out?